modules/planning/tasks/qp_spline_path/qp_spline_path_generator.cc

```diff
@@ -321,7 +320,6 @@ bool QpSplinePathGenerator::AddConstraint(const QpFrenetFrame& qp_frenet_frame,
   if (!is_change_lane_path_) {
     spline_constraint->AddPointDerivativeConstraintInRange(
         evaluated_s_.back(), 0.0, kEndPointBoundaryEpsilon);
-  }
```

Comment:
Also this one, seems unbalanced bracket?

Reply:
Thank you, fixed already.